examples/launch_flatten.cpp

```diff
@@ -89,63 +89,62 @@ int main(int RAJA_UNUSED_ARG(argc), char **RAJA_UNUSED_ARG(argv[]))
 
   std::cout << "\n Running device version of teams_flatten example ...\n";
 
   RAJA::ReduceSum<reduce_policy, int> device_kernel_sum(0);
   RAJA::View<int, RAJA::Layout<2>> d_A_2DView(d_A_ptr, N, N);
   RAJA::View<int, RAJA::Layout<1>> d_A_1DView(d_A_ptr, NN);
 
-  RAJA::launch<device_launch>
-    (launch_params, [=] RAJA_HOST_DEVICE (RAJA::LaunchContext ctx)
-     {
-
-       RAJA::loop<device_inner_pol1>(ctx, RAJA::RangeSegment(0, N), [&] (int j) {
-         RAJA::loop<device_inner_pol0>(ctx, RAJA::RangeSegment(0, N), [&] (int i) {
-             d_A_2DView(j, i) = i + j;
-           });
-         });
-
-       ctx.teamSync();
-
-       // RAJA flatten policy will reshape a 2/3D thread team to 1D simplifying
-       // accumulating memory contents
-       RAJA::loop<device_flatten_pol>(ctx, RAJA::RangeSegment(0, NN), [&] (int i) {
-           device_kernel_sum += d_A_1DView(i);
-       });
-
-     });
-
-//----------------------------------------------------------------------------//
+  RAJA::launch<device_launch>(
+      launch_params, [=] RAJA_HOST_DEVICE(RAJA::LaunchContext ctx) {
+        RAJA::loop<device_inner_pol1>(
+            ctx, RAJA::RangeSegment(0, N), [&](int j) {
+              RAJA::loop<device_inner_pol0>(ctx,
+                                            RAJA::RangeSegment(0, N),
+                                            [&](int i) {
+                                              d_A_2DView(j, i) = i + j;
+                                            });
+            });
+
+        ctx.teamSync();
+
+        // RAJA flatten policy will reshape a 2/3D thread team to 1D simplifying
+        // accumulating memory contents
+        RAJA::loop<device_flatten_pol>(ctx,
+                                       RAJA::RangeSegment(0, NN),
+                                       [&](int i) {
+                                         device_kernel_sum += d_A_1DView(i);
+                                       });
+      });
+
+  //----------------------------------------------------------------------------//
 
   std::cout << "\n Running host version of teams_flatten example ...\n";
 
   RAJA::ReduceSum<reduce_policy, int> host_kernel_sum(0);
   RAJA::View<int, RAJA::Layout<2>> h_A_2DView(h_A_ptr, N, N);
   RAJA::View<int, RAJA::Layout<1>> h_A_1DView(h_A_ptr, NN);
 
-  RAJA::launch<host_launch>
-    (launch_params, [=] RAJA_HOST_DEVICE (RAJA::LaunchContext ctx)
-    {
-
-       RAJA::loop<host_loop>(ctx, RAJA::RangeSegment(0, N), [&] (int j) {
-         RAJA::loop<host_loop>(ctx, RAJA::RangeSegment(0, N), [&] (int i) {
-             h_A_2DView(j, i) = i + j;
-           });
-         });
-
-       ctx.teamSync();
+  RAJA::launch<host_launch>(
+      launch_params, [=] RAJA_HOST_DEVICE(RAJA::LaunchContext ctx) {
+        RAJA::loop<host_loop>(ctx, RAJA::RangeSegment(0, N), [&](int j) {
+          RAJA::loop<host_loop>(ctx, RAJA::RangeSegment(0, N), [&](int i) {
+            h_A_2DView(j, i) = i + j;
+          });
+        });
```

Comment:
This looks good though, not sure why the previous one got shifted around so much.